designs/resource-properties.md

```diff
@@ -287,52 +287,6 @@ structure PipelineDescription {
 * Member must target a structure shape, and members of that structure shape may
   not use @notProperty
 
-### Option 2: Associating top level resource properties to structures with `@associatedResource`
-
-In order to support existing services migration to resource properties we will
-introduce a new trait named `@associatedResource` to associate a structure shape
-with the top level properties of a specific resource instead of a top level
-lifecycle operation input or output. This trait can only be allowed to structure
-shapes one below the input or output of a resource lifecycle operation.
-
-```
-resource Pipeline {
-...
-     properties: {
-        name: String
-        foo: Baz
-        rank: Integer
-    },
-    create: CreatePipeline
-    ...
-}
-
-operation CreatePipeline {
-    input := CreatePipelineInput
-}
-
-structure CreatePipelineInput {
-     pipeline: PipelineDescription
-}
-
-@associatedResource("Pipeline")
-structure PipelineDescription {
-     name: String
-    foo: Baz
-    @property(name:"rank")
-    ranking: Integer
- }
-```
-
-#### `@` associatedResource trait definition
-
-The `@associatedResource` trait is defined in Smithy’s prelude as:
-
-```
-@trait(selector: "structure") //TODO: How to force off of top level input/output
-string associatedResource
-```
-
 ### Tracking append vs replace with `@cfnCollection`
```

Comment:
Is this nested under `Addressing properties root misalignment with operation input/output`? Why?

Reply:
If so, it was by mistake. The entire section is gone now.